docs/channels.md

```diff
@@ -0,0 +1,760 @@
+# Channels
```

Comment:
We probably should have some kind of introductory paragraph about what channels is, what kinds of posts we have and how those are displayed. @jamesacklin wrote some stuff up in the readme, maybe can help with this. This can wait until later though just leaving a note here.